mllib/src/main/scala/org/apache/spark/ml/clustering/LDA.scala

```diff
@@ -468,7 +452,37 @@ class LDAModel private[ml] (
 /**
  * :: Experimental ::
  *
- * Distributed model fitted by [[LDA]] using Expectation-Maximization (EM).
+ * Local (non-distributed) model fitted by [[LDA]].
+ *
+ * This model stores the inferred topics only; it does not store info about the training dataset.
+ */
+@Since("1.6.0")
+@Experimental
+class LocalLDAModel private[ml] (
+    uid: String,
```

Comment:
Do you mean the Since annotations?  It's because those constructor args are also vals which will appear in the public API.